app/models/communication.rb

```diff
@@ -58,6 +60,8 @@ def cant_update_if_published
   end
 
   def send_notification
+    return false if recurrent_on
```

Comment:
Just change the condition on the `after_save` call above instead of adding multiple conditions in different places